React/Base/RCTComponentEvent.h

```diff
@@ -0,0 +1,19 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affilities.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+#import <React/RCTEventDispatcher.h>
+
+/**
+ * Generic untyped event for Components. Used internally by RCTDirectEventBlock and
+ * RCTBubblingEventBlock, for other use cases prefer using a class that implements
+ * RCTEvent to have a type safe way to initialize it.
+ */
+@interface RCTComponentEvent : NSObject<RCTEvent>
+
+- (instancetype)initWithName:(NSString *)name viewTag:(NSNumber *)viewTag body:(NSDictionary *)body;
```

Comment:
We need to add some nonnull attributes to prevent crash?

Reply:
This file could use `NS_ASSUME_NONNULL_BEGIN`, feel free to open a PR :)